src/app/GitCommands/Git/GitModule.cs

```diff
@@ -1779,7 +1779,8 @@ public bool StageFile(string file)
                 {
                     "--add",
                     file.ToPosixPath().Quote()
-                });
+                },
+                throwOnErrorExit: false);
```

Comment:
As the result of `StageFile` is never evaluated, rather remove the return value and throw.
If there are known uncritical exit codes, please handle them explicitly.

Reply:
OK, changes requires changes of the plugin interface though.